### DISCUSSION TITLE
Can't get yande.re to work

### DISCUSSION THREAD

**Discussion by izaz4141** [end-4/dots-hyprland#676] [Q&A] [answered]
I can't seem to get yande.re to work it either gives me 
`Gio.TlsError: Unacceptable TLS certificate`
or
`Gio.IOErrorEnum: Could not connect to yande.re: Socket I/O timed out`

when I tried /mode konachan it works but not with yandere
I tried to put the url from `${APISERVICES[this._mode].endpoint}${paramString}` to firefox and it loads the JSON just fine
I already reinstalled ca-certificates and ca-certificates-mozilla (it didn't work either)

I also tried to write gelbooru extension but it got the same problem

I'm sorry I really don't know what's going on

**Comment by end-4** [comment]
unfortunately idk either
i'm just an endeavouros noob, not arch user

**Comment by izaz4141** [comment, marked as answer]
Turns out it just that my internet provider blocked them
I guess I'll just use systemd dns

**Reply by end-4** [reply]
alright
we have a button to enable cloudflare's 1.1.1.1 on the right sidebar btw (will be shown if you set it up)
